modules/install/Classes/Import.php

```diff
@@ -179,9 +179,13 @@ public function ImportXML($rewrite = null)
                                 $default = 'default CURRENT_TIMESTAMP';
                                 $default_xml = 'CURRENT_TIMESTAMP';
                                 $extra = 'on update CURRENT_TIMESTAMP';
-                            } elseif ($type == 'datetime' or $type == 'date' or $type == 'time' or $type == 'blob') {
-                                $default = '';
-                            } elseif ($type == 'text' or $type == 'tinytext' or $type == 'mediumtext' or $type == 'longtext') {
+                            } elseif ($type == 'datetime' or $type == 'date' or $type == 'time') {
+                                if ($default_xml != '') {
+                                    $default = "default '$default_xml'";
+                                } else {
+                                    $default = '';
+                                }
```

Comment:
```suggestion
                                $default = '';
                                if ($default_xml != '') {
                                    $default = "default '$default_xml'";
                                }
```
This way we would not require an `else` part.